diff -r -u editor-on-fire-r1437_orig/src/main.c editor-on-fire-r1437/src/main.c
--- editor-on-fire-r1437_orig/src/main.c	2015-08-15 22:39:01.000000000 +0200
+++ editor-on-fire-r1437/src/main.c	2015-10-29 21:41:24.285928638 +0100
@@ -4107,30 +4107,22 @@
 			allegro_message("MP3 support disabled.\n\nPlease install LAME and Vorbis Tools if you want MP3 support.");
 		}
 	#elif defined(ALLEGRO_MACOSX)
-		if((eof_system("lame -S check.wav >foo") == 0) && (eof_system("oggenc -h >foo") == 0))
+		if((eof_system("lame --help >/dev/null") == 0) && (eof_system("oggenc -h >/dev/null") == 0))
 		{
 			eof_supports_mp3 = 1;
-			(void) delete_file("check.mp3");
-			(void) delete_file("foo");
 		}
 		else
 		{
 			eof_supports_mp3 = 0;
-			(void) delete_file("foo");
-			allegro_message("MP3 support disabled.\n\nPlease install LAME and Vorbis Tools if you want MP3 support.");
 		}
 	#else
-		if((eof_system("lame -S check.wav >foo") == 0) && (eof_system("oggenc -h >foo") == 0))
+		if((eof_system("lame --help >/dev/null") == 0) && (eof_system("oggenc -h >/dev/null") == 0))
 		{
 			eof_supports_mp3 = 1;
-			(void) delete_file("check.wav.mp3");
-			(void) delete_file("foo");
 		}
 		else
 		{
 			eof_supports_mp3 = 0;
-			(void) delete_file("foo");
-			allegro_message("MP3 support disabled.\n\nPlease install LAME and Vorbis Tools if you want MP3 support.");
 		}
 	#endif
 	}
