diff -ru MEGAsync_orig/src/MEGASync/mega/src/posix/net.cpp MEGAsync/src/MEGASync/mega/src/posix/net.cpp
--- MEGAsync_orig/src/MEGASync/mega/src/posix/net.cpp	2016-09-04 00:57:26.709821499 +0200
+++ MEGAsync/src/MEGASync/mega/src/posix/net.cpp	2016-09-04 01:00:18.596739533 +0200
@@ -34,7 +34,7 @@
     std::transform(curlssl.begin(), curlssl.end(), curlssl.begin(), ::tolower);
 
 #if !defined(USE_CURL_PUBLIC_KEY_PINNING) || defined(WINDOWS_PHONE)
-    if (!strstr(curlssl.c_str(), "openssl"))
+    if (!strstr(curlssl.c_str(), "openssl") && !strstr(curlssl.c_str(), "libressl"))
     {
         LOG_fatal << "cURL built without OpenSSL support. Aborting.";
         exit(EXIT_FAILURE);
