diff -ru darling-9999_orig/src/setup-ld-so.sh darling-9999/src/setup-ld-so.sh
--- darling-9999_orig/src/setup-ld-so.sh	2017-01-18 00:31:21.684299515 +0100
+++ darling-9999/src/setup-ld-so.sh	2017-01-18 01:17:01.591951891 +0100
@@ -6,7 +6,6 @@
 for file in /etc/ld.so.conf $(find /etc/ld.so.conf.d/ -type f); do
     # Copy lines from e.g. /etc/ld.so.conf into ./etc/ld.so.conf,
     # prepending "/Volumes/SystemRoot" to each line that starts with a slash
-    awk '/^\// { print "/Volumes/SystemRoot" $0 }; /^[^\/]/' $file > .$file
+    awk '/^\// { print "/Volumes/SystemRoot" $0 }; /^[^\/]/' $file > ${PORTAGE_INSTDIR}/usr/libexec/darling/$file
 done
 
-unshare --mount bash -c "mount --rbind / Volumes/SystemRoot && ldconfig -r . -X"
