diff -ru darling_orig/src/setup-ld-so.sh darling/src/setup-ld-so.sh
--- darling_orig/src/setup-ld-so.sh	2017-01-26 13:07:11.447614854 +0100
+++ darling/src/setup-ld-so.sh	2017-01-26 13:14:29.067800033 +0100
@@ -1,7 +1,6 @@
 #! /bin/bash
 
-# This runs as root,
-# with the current directory set to ${CMAKE_INSTALL_PREFIX}/libexec/darling
+exit
 
 for file in /etc/ld.so.conf $(find /etc/ld.so.conf.d/ -type f); do
     # Copy lines from e.g. /etc/ld.so.conf into ./etc/ld.so.conf,
