diff -ru darling_orig/src/CMakeLists.txt darling/src/CMakeLists.txt
--- darling_orig/src/CMakeLists.txt	2017-04-06 20:21:04.467890213 +0200
+++ darling/src/CMakeLists.txt	2017-04-06 20:21:19.941564429 +0200
@@ -189,10 +189,3 @@
 add_subdirectory(external/libcxxabi)
 add_subdirectory(external/libcxx)
 
-#######################
-# Linux kernel module #
-#######################
-
-add_custom_target(lkm
-	COMMAND gmake
-	WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/lkm")
