diff -ru package_orig/test/test-runfiles.js package/test/test-runfiles.js
--- package_orig/test/test-runfiles.js	2015-03-03 05:09:22.000000000 +0100
+++ package/test/test-runfiles.js	2016-08-19 22:03:52.033319794 +0200
@@ -116,8 +116,8 @@
     var dir2 = __dirname + '/fixtures/dir2';
 
     // git doesn't like empty directories, so we have to create one
-    path.exists(dir2, function (exists) {
-        if (!exists) {
+    fs.access(dir2, function (err) {
+        if (err) {
             fs.mkdirSync(dir2, 0777);
         }
 
