diff -ru megatools-1.9.98_orig/lib/mega.c megatools-1.9.98/lib/mega.c
--- megatools-1.9.98_orig/lib/mega.c	2017-02-15 17:22:45.706557789 +0100
+++ megatools-1.9.98/lib/mega.c	2017-02-15 17:24:58.552834369 +0100
@@ -803,7 +803,7 @@
     return FALSE;
   }
 
-#if OPENSSL_VERSION_NUMBER >= 0x10100000L
+#if OPENSSL_VERSION_NUMBER >= 0x10100000L && !defined(LIBRESSL_VERSION_NUMBER)
   RSA_get0_key(key, &k->m, &k->e, &k->d);
   RSA_get0_factors(key, &k->q, &k->p);
   RSA_get0_crt_params(key, NULL, NULL, &k->u);
@@ -2988,7 +2988,7 @@
   GByteArray* buffer;
 };
 
-#if OPENSSL_VERSION_NUMBER >= 0x10100000L
+#if OPENSSL_VERSION_NUMBER >= 0x10100000L && !defined(LIBRESSL_VERSION_NUMBER)
 void AES_ctr128_encrypt(
   const unsigned char *in, 
   unsigned char *out,
diff -ru megatools-1.9.98_orig/lib/tools.c megatools-1.9.98/lib/tools.c
--- megatools-1.9.98_orig/lib/tools.c	2017-02-15 17:22:45.706557789 +0100
+++ megatools-1.9.98/lib/tools.c	2017-02-15 17:25:30.670175738 +0100
@@ -107,7 +107,7 @@
   { NULL }
 };
 
-#if OPENSSL_VERSION_NUMBER >= 0x10100004L
+#if OPENSSL_VERSION_NUMBER >= 0x10100004L && !defined(LIBRESSL_VERSION_NUMBER)
 static void init_openssl_locking()
 {
   // OpenSSL >= 1.1.0-pre4 doesn't require specific callback setup
@@ -567,7 +567,7 @@
   CRYPTO_cleanup_all_ex_data();
   ERR_free_strings();
 
-#if OPENSSL_VERSION_NUMBER < 0x10100004L
+#if OPENSSL_VERSION_NUMBER < 0x10100004L || defined(LIBRESSL_VERSION_NUMBER)
   CRYPTO_set_id_callback(NULL);
   CRYPTO_set_locking_callback(NULL);
 #endif
