diff -ru citra_orig/CMakeLists.txt citra/CMakeLists.txt
--- citra_orig/CMakeLists.txt	2017-06-01 00:39:32.670951477 +0200
+++ citra/CMakeLists.txt	2017-06-01 00:41:33.058545744 +0200
@@ -5,6 +5,8 @@
 
 project(citra)
 
+option(USE_SYSTEM_BOOST "Use system-wide boost libraries" OFF)
+
 option(ENABLE_SDL2 "Enable the SDL2 frontend" ON)
 option(CITRA_USE_BUNDLED_SDL2 "Download bundled SDL2 binaries" OFF)
 
@@ -152,13 +154,12 @@
     message(STATUS "libpng not found. Some debugging features have been disabled.")
 endif()
 
-find_package(Boost 1.63.0 QUIET)
-if (NOT Boost_FOUND)
-    message(STATUS "Boost 1.63.0 or newer not found, falling back to externals")
-
-    set(BOOST_ROOT "${CMAKE_SOURCE_DIR}/externals/boost")
-    set(Boost_NO_SYSTEM_PATHS OFF)
-    find_package(Boost QUIET REQUIRED)
+if (USE_SYSTEM_BOOST)
+	find_package(Boost 1.63.0 QUIET REQUIRED)
+else()
+	set(BOOST_ROOT "${CMAKE_SOURCE_DIR}/externals/boost")
+	set(Boost_NO_SYSTEM_PATHS OFF)
+	find_package(Boost QUIET REQUIRED)
 endif()
 
 # Prefer the -pthread flag on Linux.
