--- citra_orig/CMakeLists.txt	2018-08-25 13:09:47.269613265 +0700
+++ citra/CMakeLists.txt	2018-08-25 13:11:16.542508061 +0700
@@ -18,6 +18,8 @@
 
 option(ENABLE_WEB_SERVICE "Enable web services (telemetry, etc.)" ON)
 
+option(USE_SYSTEM_BOOST "Use system-wide boost libraries" OFF)
+
 option(ENABLE_CUBEB "Enables the cubeb audio backend" ON)
 
 option(USE_DISCORD_PRESENCE "Enables Discord Rich Presence" OFF)
@@ -182,10 +184,9 @@
 # System imported libraries
 # ======================
 
-find_package(Boost 1.66.0 QUIET)
-if (NOT Boost_FOUND)
-    message(STATUS "Boost 1.66.0 or newer not found, falling back to externals")
-
+if (USE_SYSTEM_BOOST)
+       find_package(Boost 1.66.0 QUIET REQUIRED)
+else()
     set(BOOST_ROOT "${CMAKE_SOURCE_DIR}/externals/boost")
     set(Boost_NO_SYSTEM_PATHS OFF)
     find_package(Boost QUIET REQUIRED)
