diff -ru citra_orig/CMakeLists.txt citra/CMakeLists.txt
--- citra_orig/CMakeLists.txt	2018-03-17 14:55:44.678772985 +0100
+++ citra/CMakeLists.txt	2018-03-17 15:17:37.696464953 +0100
@@ -19,6 +19,8 @@
 option(ENABLE_WEB_SERVICE "Enable web services (telemetry, etc.)" ON)
 option(CITRA_USE_BUNDLED_CURL "FOR MINGW ONLY: Download curl configured against winssl instead of openssl" OFF)
 
+option(USE_SYSTEM_BOOST "Use system-wide boost libraries" OFF)
+
 if (ENABLE_WEB_SERVICE AND CITRA_USE_BUNDLED_CURL AND WINDOWS AND MSVC)
     message("Turning off use bundled curl as msvc can compile curl on cpr")
     SET(CITRA_USE_BUNDLED_CURL OFF CACHE BOOL "" FORCE)
@@ -169,10 +171,9 @@
 # System imported libraries
 # ======================
 
-find_package(Boost 1.63.0 QUIET)
-if (NOT Boost_FOUND)
-    message(STATUS "Boost 1.63.0 or newer not found, falling back to externals")
-
+if (USE_SYSTEM_BOOST)
+	find_package(Boost 1.63.0 QUIET REQUIRED)
+else()
     set(BOOST_ROOT "${CMAKE_SOURCE_DIR}/externals/boost")
     set(Boost_NO_SYSTEM_PATHS OFF)
     find_package(Boost QUIET REQUIRED)
