diff -ru sophia-version_2.2_orig/makefile sophia-version_2.2/makefile
--- sophia-version_2.2_orig/makefile	2016-08-16 13:38:07.000000000 +0200
+++ sophia-version_2.2/makefile	2016-08-18 20:25:55.497506781 +0200
@@ -11,10 +11,9 @@
 endif
 
 CFLAGS_DEBUG   = -DSS_INJECTION_ENABLE -g
-CFLAGS_OPT     = -O2
 CFLAGS_STRICT  = -std=c99 -pedantic -Wextra -Wall
 CFLAGS_MISC    = -Wno-unused-function -fPIC -fno-stack-protector -fvisibility=hidden
-SOPHIA_CFLAGS  = $(CFLAGS_DEBUG) \
+SOPHIA_CFLAGS  = \
                  $(CFLAGS_OPT) \
                  $(CFLAGS_STRICT) \
                  $(CFLAGS_COVERAGE) \
@@ -24,7 +23,6 @@
 SOPHIA_DEP     = $(wildcard sophia/*/*) makefile
 SOPHIA_VMAJOR  = 2
 SOPHIA_VMINOR  = 2
-SOPHIA_BUILD   = `git rev-parse --short HEAD`
 
 all: banner static dynamic
 banner:
@@ -45,12 +43,6 @@
 	@$(CC) sophia.o $(SOPHIA_LDFLAGS) -o libsophia.so.2.2.0
 	@ln -sf libsophia.so.2.2.0 libsophia.so.2.2
 	@ln -sf libsophia.so.2.2.0 libsophia.so
-ifeq ($(shell uname), Linux)
-	@strip --strip-unneeded libsophia.so.2.2.0
-endif
-ifeq ($(shell uname), Darwin)
-	@strip -u -r -x libsophia.so.2.2.0
-endif
 static: libsophia.a
 dynamic: libsophia.so
 clean:
diff -ru sophia-version_2.2_orig/test/makefile sophia-version_2.2/test/makefile
--- sophia-version_2.2_orig/test/makefile	2016-08-16 13:38:07.000000000 +0200
+++ sophia-version_2.2/test/makefile	2016-08-18 20:34:59.334864868 +0200
@@ -72,7 +72,7 @@
 else
 STS_COVERAGE   =
 endif
-STS_CFLAGS     = $(SOPHIA_INCLUDE) -D_GNU_SOURCE=1 -g -O0 -Wall -I. -I../sophia -Isuite/ -pthread
+STS_CFLAGS     = $(SOPHIA_INCLUDE) -D_GNU_SOURCE=1 -Wall -I. -I../sophia -Isuite/ -pthread $(CFLAGS_OPT)
 STS_LDFLAGS    = ../sophia.o -pthread -lm -lrt $(STS_COVERAGE)
 STS_OBJECTS    = suite/st_r.o \
                  suite/st_suite.o \
