diff -ru cpu_features-0.2.0_orig/CMakeLists.txt cpu_features-0.2.0/CMakeLists.txt
--- cpu_features-0.2.0_orig/CMakeLists.txt	2018-09-07 15:54:51.000000000 +0200
+++ cpu_features-0.2.0/CMakeLists.txt	2018-09-07 20:30:27.804826708 +0200
@@ -90,12 +90,6 @@
   # Automatically incorporate googletest into the CMake Project if target not
   # found.
   if(NOT TARGET gtest OR NOT TARGET gmock_main)
-    # Download and unpack googletest at configure time.
-    configure_file(
-      cmake/googletest.CMakeLists.txt.in
-      googletest-download/CMakeLists.txt
-    )
-
     execute_process(
       COMMAND ${CMAKE_COMMAND} -G "${CMAKE_GENERATOR}" .
       RESULT_VARIABLE result
