diff -ru Sachesi-2.0.4_orig/src/installer_establish.cpp Sachesi-2.0.4/src/installer_establish.cpp
--- Sachesi-2.0.4_orig/src/installer_establish.cpp	2018-05-12 16:35:19.000000000 +0200
+++ Sachesi-2.0.4/src/installer_establish.cpp	2018-05-14 16:47:50.879893363 +0200
@@ -18,7 +18,7 @@
 #include "installer.h"
 
 // Compatibility for old OpenSSL
-#if OPENSSL_VERSION_NUMBER < 0x10100005L
+#if OPENSSL_VERSION_NUMBER < 0x10100005L || defined(LIBRESSL_VERSION_NUMBER)
 static void RSA_get0_key(const RSA *r, const BIGNUM **n, const BIGNUM **e, const BIGNUM **d)
 {
     if(n != NULL)
